packages/fasm/KagamiBuild

```diff
@@ -0,0 +1,15 @@
+# Description: An assembler for x86 processors.
+# URL:         https://flatassembler.net
+# Maintainer:  ixavoz, ixavoz at disroot dot com
+# Section:     devel
+name=fasm
+version=1.73.22
+release=1
+source=("https://flatassembler.net/fasm-$version.tgz")
+build()
```

Comment:
```
build() {
```
not
 ```
build()
{
```

Reply:
It's my style. Fix for consistency to the rest of KagamiBuild files anyway.